src/app/admin/mod.rs

```diff
@@ -77,6 +77,35 @@ where
     }
 }
 
+impl<M: FmtMetrics + Clone + Send + 'static> svc::Service<tls::Connection> for Accept<M> {
```

Comment:
I find it kind of odd that use two different types of `Service` here that seem to have the same purpose. Any reason `Admin` implements `hyper::Service` and not `tower`?

Reply:
I don't know. Unrelated to this change, though.